docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -13,7 +13,27 @@ Integrate your ESP32-based device to manage it via Coiote DM.
 0. Create a project directory for the integration.
 0. Go to [https://github.com/AVSystem/Anjay-esp32-client/releases](https://github.com/AVSystem/Anjay-esp32-client/releases) and download `m5stickc-plus.bin` and `nvs_partition_gen.py` to your project directory.
 
-## Step 2: Configure the client using an NVS partition
+## Step 2: Add device to Coiote DM
+
+To connect your M5StickC to the Coiote IoT Device Management LwM2M Server, use your access to a Coiote DM installation, or register at https://eu.iot.avsystem.cloud/ to get access.
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** field, type the endpoint name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Security mode** section, select the **PSK** mode.
+    - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key identity** field, type the identity provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key** field, type the `psk` key provided in the `nvs_config.csv`.
+    ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+0. In the **Connect your device** step, the server is waiting for the board to connect. You can now start connecting the device.
+
+## Step 3: Configure the client using an NVS partition
 
 0. Create a `nvs_config.csv` file and save it in your project directory. In the file, provide your credentials in [wifi_ssid], [wifi_password], [endpoint_name], [identity], [psk], [lwm2m_server_uri] (without the `[]` brackets). Use the following snippet as a template:
```

Comment:
Note: that file can also be downloaded from [https://github.com/AVSystem/Anjay-esp32-client/releases](https://github.com/AVSystem/Anjay-esp32-client/releases)